### PR TITLE
Set a width on TabbedCard in Storybook using same method as InfoCard

### PR DESCRIPTION
**Pull request by hooloovooo**
## Hey, I just made a Pull Request!

Fix for #564 

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

2020-04-16-101809_854x762_scrot

### CODE REVIEW

File: packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx [hunk 1]
```diff
@@ -15,19 +15,37 @@
...
 import React, { useState } from 'react';
 import { TabbedCard, CardTab } from '.';
+import { Grid } from '@material-ui/core';
+
+const cardContentStyle = { height: 200, width: 500 };
 
 export default {
   title: 'Tabbed Card',
   component: TabbedCard,
+  decorators: [
```
Comment (stefanalund) [inline packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx:25]: Oh, that's interesting. Perhaps we want to add this to the global scheme?

**Review by stefanalund — APPROVED** [review]

File: packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx (hunk 1 above)
Comment (hooloovooo) [reply packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx:25]: That could be a good idea unless there are exceptions to components that should be wrapped in a grid.

**Review by Rugvip — APPROVED** [review]
🚀

File: packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx (hunk 1 above)
Comment (stefanalund) [reply packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx:25]: Typically all components are laid in some form of grid inside Plugins. Should be fine to add to all. In a separate PR though.

**Pull request merged**